README.md

```diff
@@ -18,6 +18,10 @@ Enjoying Panel? Show your support with a [Github star](https://github.com/holovi
 <table>
 <tbody>
 <tr>
+<td>Downloads</td>
```

Comment:
Can we add the conda downloads of 170k too?

Reply:
Yep why not. The conda badge on pyviz.org is actually manually built when the site is built (see this script https://github.com/pyviz/pyviz.org/blob/master/tools/conda_downloads.py). It's available at this URL for Panel: https://pyviz.org/_static/cache/panel_conda_downloads_badge.svg.

Reply:
I'm leaning towards keeping them separate, mainly because I don't know how and I don't want this small change to take up too much time

Also, I think people can do some quick mental math to combine those numbers
<img width="826" alt="image" src="https://github.com/holoviz/panel/assets/15331990/7915ca95-f2b1-4912-a76a-d15b7004b1d3">

Reply:
+1, certainly as a first cut.

Reply:
Just a quick look at the screenshot. It seems to lack a space between the badges, and maybe change downloads to PyPi downloads or something like that.

Reply:
The badge label is default to Downloads; not sure how to change it.

Reply:
<img width="992" alt="image" src="https://github.com/holoviz/panel/assets/15331990/53352cf3-721f-4954-854e-cdf5f838eb00">